src/main/java/frc/robot/subsystems/Climber2020.java

```diff
@@ -30,13 +30,15 @@ public boolean isEnabled() {
     }
 
     public void up() {
-        if (this.isEnabled())
-            climberMotor.set(Constants.CLIMBER2020_UP_SPEED);
+        if (this.isEnabled()) {
```

Comment:
You can simply write `isEnabled()` instead of `this.isEnabled`. Follows elsewhere in this file, too.